shell/browser/ui/views/win_caption_button_container.cc

```diff
@@ -132,10 +122,7 @@ void WinCaptionButtonContainer::AddedToWidget() {
   widget_observation_.Observe(widget);
 
   UpdateButtons();
-
-  if (frame_view_->window()->IsWindowControlsOverlayEnabled()) {
```

Comment:
This will always be enabled as it's a precondition for `WinCaptionButtonContainer` being created.